samples/dashboards/component-options/sync-highlight/demo.js

```diff
@@ -0,0 +1,100 @@
+const csvData = document.getElementById('csv').innerText;
+
+const chartOptions = {
```

Comment:
If this is a sync-highlight demo, is there a need for a charts' titles? and does the drag& drop module need to be included? I'm just thinking that the demo should just show a single option, and there is a combination of some, and it is not clear, what this demo shows.